src/cli/node/with_node.rs

```diff
@@ -580,6 +593,47 @@ pub enum Command {
         #[structopt(long = "show-dr", help = "Print data request")]
         print_data_request: bool,
     },
+    #[structopt(
+        name = "searchRequests",
+        about = "Search data requests with a specific bytecode"
+    )]
+    SearchRequests {
+        /// Socket address of the Witnet node to query
+        #[structopt(short = "n", long = "node")]
+        node: Option<SocketAddr>,
+        /// init
```

Comment:
init or start? :P

Reply:
Not sure, this is copied from another command

Reply:
I think it doesnt matter, but we should use the same

Reply:
Ok, removed this comment, because it was redundant anyway.